style.css

```diff
@@ -0,0 +1,82 @@
+*{
+  font-family: sans-serif;
+}
+
+body {
+  margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+  background-color: #232323;
+}
+
+.calculator {
+  background-color: #101010;
+  width: 370px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  border: 4px solid #ffffff;
+}
+
+input {
+  height: 70px;
+  background-color: #202020;
+  padding: 0 14px;
+  width: 333px;
+}
+
+#screen {
+  font-size: 28px;
+  border: none;
+  outline: none;
+  text-align: right;
+}
+
+#screen::placeholder {
+  color: #ffffff;
+}
+
+#result {
+  border: none;
+  font-size: 25px;
+  text-align: right;
+  text-align: right;
+}
+
+.keys {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  width: 98%;
+  margin: 0.8em auto;
+}
+
+button {
+  width: 75px;
+  height: 75px;
+  border-radius: 5px;
+  outline: none;
+  border: none;
+  cursor: pointer;
+  color: #ffffff;
+}
+
+.key {
+  font-size: 0.9em;
+  background-color: #202020;
+}
+
+.key:hover {
+  transition: all 0.2s linear;
+  background-color: #303030;
+}
+
+.char {
```

Comment:
What does `char` stand for?

Reply:
**Char** stand for character

Reply:
Try using "key" as your class name instead. That should be more fitting to the usage here.

Reply:
I already used **key** for the numbers, may be I'll use **symbolKey** instead

Reply:
Fixed